### DISCUSSION TITLE
Doosan Robot

### DISCUSSION THREAD

**Discussion by HenriBenz** [visose/Robots#167] [Q&A] [answered]
Hi visose,

Our Institute is looking into purchasing a **Doosan Robot** (DR). 
Main interest lays in the DR's having a much higher breaking power, 
hence holding it's position in comparison to the cobots of competitors. 

How do you see the chances for implementing DR into the robots library?
Would you be able to assist in writing the code for the postprocessor?
Which files need to be provided beside .3dm and .xml?

Thank you for your help.
Kind Regards

**Comment by visose** [comment, marked as answer]
Hi @HenriBenz, if you need a new robot manufacturer added to the plugin, you can contract me for this work. I will then study the feasibility of integrating it in the plugin and give you an estimate based on the expected amount of work. You can email me if you are interested in pursuing this.

Some co-bots are not expected to be programmed as conventional industrial robots and might not be suitable to the workflow that this plugin is designed for. They might also require to make special components for unique behaviors, or have to special case existing functionality that is otherwise shared between existing manufacturers. But I don't know yet if this is the case with this robot.
